ci/zkasm-result.py

```diff
@@ -0,0 +1,74 @@
+import os
+import csv
+import sys
+
+# Define the directories
+tests_dir = 'cranelift/zkasm_data/spectest/i64'
+generated_dir = 'cranelift/zkasm_data/spectest/i64/generated'
+state_csv_path = 'cranelift/codegen/src/isa/zkasm/docs/state.csv'
+
+def check_compilation_status():
+    status_map = {}
+    for file in os.listdir(tests_dir):
+        if file.endswith('.wat'):
+            test_name = os.path.splitext(file)[0]
+            zkasm_file = f'{test_name}.zkasm'
+            status_map[test_name] = 'compilation success' if zkasm_file in os.listdir(generated_dir) else 'compilation failed'
```

Comment:
You can do `os.path.exists(os.path.join(generated_dir, zkasm_file))` to be more specific